tests/test_v6.py

```diff
@@ -143,6 +144,9 @@ async def client_mock(param: Param):
             )
         else:
             raise Exception
+        respx.get(f"{param.base}/recordings/list").respond(
+            json=RECORDINGS_LIST
+        )
```

Comment:
You are not actually testing that this data can be read anywhere.

Reply:
I fixed it by testing selected value combinations